app/src/test/java/mollie/tictactoe/GameHelperTest.java

```diff
@@ -0,0 +1,143 @@
+package mollie.tictactoe;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+
+public class GameHelperTest {
+
+    private GameHelper mGameHelper;
+
+    @Before
+    public void setUp() {
+        mGameHelper = new GameHelper();
+    }
+
+    @Test
+    public void createsAGameWithTwoHumanPlayers() {
+        GameEngine game = mGameHelper.createGame();
+
+        Player playerOne = game.getCurrentPlayer();
+        game.play(1);
+        Player playerTwo = game.getCurrentPlayer();
+
+        assertEquals(MobilePlayer.class, playerOne.playerType());
```

Comment:
@nickbdyer this is again a headache from my core.  It's an interface method that was used by DelayPlayer wrapper that I needed to return the class of the player it wrapped.  It shouldn't have made it into the interface or here, but it has...